vars/pythonSetupPyPipeline.groovy

```diff
@@ -215,6 +215,26 @@ def call(Map pipelineParams) {
                                 pipelineParams.dockerRepo, moduleName, moduleVersion)
             }
           } // Deploy Docker
+
+          // Deploy the html documents to the docs branch, which when using "BitBucket Pages" allows serving the documentation directly
```

Comment:
@martinm82 There needs to be an assessment of the impact of this change for other python repositories (both GitHub and BitBucket based). Can you help here with?

Reply:
We could in fact check here whether `ghp-import` is available, if not skip this stage and dump a warning that no docs will be generated due to a repo that is not aligned with the cookiecutter-python template anymore and that the user should re-run against it.

Reply:
I added a condition isDeployDocsPluginInstalled() (line 228) to check if ghp-import is available and skip the stage with a warning if not available.